src/renderer/pages/Assets/Assets/components/AssetsList/AssetsList.tsx

```diff
@@ -41,7 +42,9 @@ export const AssetsList = () => {
       const isDisabled = networkUtils.isDisabledConnection(connections[c.chainId]);
       const hasMultiPallet = !isMultisig || networkUtils.isMultisigSupported(c.options);
 
-      return !isDisabled && hasMultiPallet;
+      const hasChainAccount = activeAccounts.some((a) => accountUtils.isChainIdMatch(a, c.chainId));
```

Comment:
Suggest to create a map `{ [chainId]: true / false }` and check it inside `filteredChains` for each chain
For instance PV wallet with 300 accs would be looked up `chains x 300` times